src/installer/corehost/cli/host_interface.h

```diff
@@ -58,6 +59,7 @@ struct host_interface_t
     const pal::char_t* host_info_host_path;
     const pal::char_t* host_info_dotnet_root;
     const pal::char_t* host_info_app_path;
+    const bundle::info_t *app_bundle;
```

Comment:
`bundle::info_t` does not look to be plain data, so using it cross-binary like this where hostpolicy reads the class allocated by hostfxr could lead to compat issues.

Reply:
Thanks @elinor-fung . I've fixed this in https://github.com/dotnet/runtime/pull/34274/commits/12b63bae12ab435120b8a9b56165a86d6563406f